portfolio/src/main/webapp/script.js

```diff
@@ -54,4 +54,16 @@ document.getElementById("storyBox").style.visibility = 'visible';
 */ 
 function getRandomItemFromArray(array) {
     return array[Math.floor(Math.random() * (array.length))];
+}
+
+/**
+ * Fetch information from the 'data' servlet.
+ */
+function fetchFromData() {
+    var newDiv = document.createElement('div');
+    fetch('/data').then(response => response.text()).then((newtexT) => {
```

Comment:
What do you expect to be fetched from Data? I don't see any backend changes. Is there something built in in the project?

Reply:
I see that you have a servlet that responds with HelloWorld. 
Will you change and re-deploy it on later stages?

Reply:
Yes, it is something that was built in in the project, that we didn't really change yet. 
The tutorial says: 
"""After completing the fetch walkthrough, your portfolio page should use the fetch() function to request a message from the servlet, and then add that message to the DOM after the page loads or after the user clicks a button. It's okay if the message is still a hard-coded "Hello [your name]" string.
It might not feel like you're implementing a comments feature yet, but engineers often work in small steps like this to confirm that each piece works and that everything is connected. Make sure you run a development server to make sure everything works.
After confirming that everything works, create a pull request from what you have so far, and send that to your host for a code review! """

Reply:
Ack, thanks for the reference. 

Note: to create a citation in Markdown, add ">" before any line, like so: 
> A smart man once said...